syntax = "proto3";
package jobs;




import "ops_automation_sdk/model/ops_automation/bind_resource.proto";
import "ops_automation_sdk/model/ops_automation/mail_info.proto";

/*
UpdateJobs返回
*/
message UpdateJobsResponse {
    /*
    作业id
    */
    string id = 1 ;
}

/*
UpdateJobsApi返回
*/
message UpdateJobsResponseWrapper {
    /*
    返回码
    */
    int32 code = 1 ;
    /*
    返回码解释
    */
    string codeExplain = 2 ;
    /*
    错误详情
    */
    string error = 3 ;
    /*
    返回数据
    */
    jobs.UpdateJobsResponse data = 4 ;
}

